library/templates/dcp_boroboundaries_wi.yml

```diff
@@ -29,4 +29,8 @@ dataset:
       ### ESRI Shapefile
       The borough boundaries of New York City including portions of the borough under water.
     url: https://www1.nyc.gov/site/planning/data-maps/open-data/districts-download-metadata.page
-    dependents: []
\ No newline at end of file
+    dependents: 
```

Comment:
my understanding of dependents are things that this product update is depending on. But I guess we could have our own definition for dependents as well.